app/src/test/java/ingsw/group1/findmyphone/log/LogManagerTest.java

```diff
@@ -26,8 +33,10 @@
 @RunWith(RobolectricTestRunner.class)
 public class LogManagerTest {
 
-    private static final int TEST_LOG_SIZE = 100;
+    private static final String DB_NAME = LogManager.DEFAULT_LOG_DATABASE;
+    private static final int TEST_LOG_SIZE = 10;
 
+    private SMSContactManager contacts;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'contacts' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5082269)